lib/support/GallerySetupUtilities.class

```diff
@@ -362,12 +362,12 @@ class GallerySetupUtilities {
 	 * attack.  Either create a clean session, or if we detect that this session is clean,
 	 * sign it in a way that we'll recognize.
 	 */
-	$remoteId = GallerySession::getRemoteIdentifier();
```

Comment:
I don't think we should need this change - `getRemoteIdentifier()` is already a static method, and I didn't see an error message about this, so I think we can just revert this line.

Reply:
lgtm.  i'm more comfortable in perforce, so it is not immediately obvious hot to revert, or should I just abandon the change request?

Reply:
Yeah, I'm not very experienced at GitHubbing :-) but I think the usual practice is just to abandon the pull request (and delete your branch, if you want) - since it hasn't been merged in, no code changes have been made yet, so you can just walk away.